src/components/ExploreNav/styled.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import { BLACK, H3, CREAM, PINK } from "../../styles/styles";
+import { min } from "../../styles/breakpoints";
+import Cursor from "../../images/cursor-pointer.svg";
+import { AnchorLink } from "gatsby-plugin-anchor-links";
+
+export const StyledLink = styled.div`
+  white-space: nowrap;
+  height: 3rem;
+  width: 100%;
+  transform: skew(15deg);
+  background-color: transparent;
+  cursor: url(${Cursor}) 20 20, pointer;
```

Comment:
nit: import `PointerStyles` from the styles file so you don't have to import the cursor image everywhere it's used; it replaces this whole line